.readthedocs.yml

```diff
@@ -1,6 +1,41 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html
+# for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: dirhtml
+  configuration: docs/conf.py
+  fail_on_warning: true  # https://github.com/pypa/setuptools/issues/2424
+
+# Build documentation with MkDocs
+#mkdocs:
+#  configuration: mkdocs.yml
+#  fail_on_warning: true
+
+# Optionally build your docs in additional formats
+# such as PDF and ePub
+formats:
+- epub
+- pdf
+- htmlzip
+
+submodules:
+  include: all  # []
+  exclude: []
+  recursive: true
+
+build:
+  image: latest
```

Comment:
Is all of this necessary to support v2 of the config?

Reply:
Most of it is a copy-paste from cheroot. `formats` is attempting to replicate the current setup since the doc says that the default is `[]` and I see on the web UI that there's all these formats linked.

Regarding the image, `latest` seems to be the default and available Python versions depend on the image used. It's usually helpful to be explicit about such things because it's like pinned constraints for virtualenvs -- it's better to know how to reproduce stuff, and it's good if all of the details are in the same place vs trying to figure out where the defaults are documented and how the options influence each other.